src/travel_tracker_service.js

```diff
@@ -69,9 +65,43 @@ async function getTravelerDestinations(travelerTrips, traveler) {
   return { travelerTrips, traveler, travelerDestinations };
 }
 
+
+async function getAllTripsForAgent() {
+  const url =
+    "https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips";
```

Comment:
I notice in this file, almost every function has `const url` in it.

If you defined a constant `baseUrl` that was `"https://fe-apps.herokuapp.com/api/v1/travel-tracker`

Then you could do something like
```
const tripsUrl = url + '/data/trips/trips'
const destinationsUrl = 
```

etc etc